ui_framework/components/info_panel/_info_panel.scss

```diff
@@ -1,8 +1,59 @@
 .kuiInfoPanel {
-  padding: 20px;
+  padding: 14px 20px 18px;
   line-height: $lineHeight;
+  border: 2px solid;
+}
+
+/**
+ * 1. TODO: Pick a hex value instead of making these colors translucent.
```

Comment:
I'm fine with it as long as you have some way of tracking it, i.e a Github issue or some other task board you use